app/components/files_search/file_results.tsx

```diff
@@ -161,6 +164,7 @@ const FileResults = ({
                         values={{count: orderedFileInfos.length}}
                     />
                 }
+                ItemSeparatorComponent={() => <View style={styles.separator}/>}
```

Comment:
To avoid passing a new prop on every render, you can memoize this. Or since the style is constant, you could get it completely out of the component. Something like:

```
const separatorStyle = {height: 10};
const Separator = () => <View style={separatorStyle}/>;

const FileResults = ({
   ...
}) => {
   ...
       ItemSeparatorComponent={Separator}
   ...
};
```